javascript/modules/client/src/client.ts

```diff
@@ -630,6 +693,88 @@ export class Client extends ClientCore implements IClient {
     return Promise.resolve({ deposits, withdrawals });
   }
 
+  private async _buildGrantAction(
```

Comment:
If the reason for the function to be `async` is the ENS resolution, I would suggest the SDK only accepted `addresses` and have a sync function.

This could be misleading and confusing, because the proposal creator could be defining `my-wallet.eth`, resolving to `0x1234...` upon creation, then be updated to point to `0x5678...`. Which is the correct address to give permissions to?